source/includes/_endpoint-batch.md

```diff
@@ -39,20 +41,62 @@ Must contain the URL of a dataset that the current user can read. This action wi
 }
 ```
 
-If you wish to include only certain variables from the source dataset you're appending from, you can include an ``function`` in the body of the entity.  The function should be a ZCL function name.  Its arguments should be present in the ``args`` key in the body.
+Note that by default only public and not hidden variables from the dataset will
```

Comment:
Ok. That difference in behavior based on authorization on the incoming dataset is not at all clear in these docs. Please make that explicit.